src/components/Toggle/Toggle.ts

```diff
@@ -35,7 +40,6 @@ namespace fabric {
     }
 
     private _toggleHandler(): void {
-      this._toggleField.classList.toggle("is-selected");
```

Comment:
Hi, no this was a mistake. The _toggleHandler() should still stay the same. Should I create a new PullRequest?